README.md

```diff
@@ -1,2 +1,90 @@
 # open-sir
-Python SIR-x model implementation
+
+Open source pandemics modelling API in python. The models implemented are in the scope of compartmental models of disease transmission, such as the widely used [Susceptible-Infected-Removed (SIR) model](http://rocs.hu-berlin.de/corona/docs/forecast/model/#classic-sir-dynamics). Recently, a new epidemic model based in SIR, SIR-X, was developed by the [Robert Koch Institut](http://rocs.hu-berlin.de/corona/docs/forecast/model/#sir-x-dynamics-outbreaks-with-temporally-increasing-interventions) and it is also implemented in open-sir.
+
+The API features model construction, parameter fitting and calculation of confidence intervals.
+
+## Getting Started
+
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes. See deployment for notes on how to deploy the project on a live system.
+
+### Prerequisites
+
+What things you need to install the software and how to install them
+
+```
+Give examples
+```
+
+### Installing
+
+A step by step series of examples that tell you how to get a development env running
+
+Say what the step will be
+
+```
+Give the example
+```
+
+And repeat
+
+```
+until finished
+```
+
+End with an example of getting some data out of the system or using it for a little demo
+
+## Running the tests
+
+Explain how to run the automated tests for this system
+
+### Break down into end to end tests
```

Comment:
I think most of the testing stuff should be in the CONTRIBUTING.md files rather than the README.md

Reply:
This also include the coding style